pkg/roachprod/install/nodes.go

```diff
@@ -44,7 +44,9 @@ func ListNodes(s string, numNodesInCluster int) (Nodes, error) {
 		return nil, errors.AssertionFailedf("invalid number of nodes %d", numNodesInCluster)
 	}
 
-	if s == "all" {
+	// "L" is a special value that also returns all nodes, but implies a load
+	// balancer should be used when applicable.
+	if s == "all" || s == "L" {
```

Comment:
Why not `LB`? `L` doesn't seem like a memorable mnemonic unlike `all`. Should it also be case-insensitive?

Reply:
I matched this to how the expanders also select the load balancer vs. node IPs. I can update both. I'll maybe add it to the command help as well to make it clear as an option.